### DISCUSSION TITLE
File upload

### DISCUSSION THREAD

**Discussion by FELS-Zak** [kimai/kimai#4194] [Questions & Answers]
is it possible for a certain user role ( a custom added one ) to be able to upload a file when submitting a time sheet but only monthly. 
           I want consultants to add their invoice in. Or is there a better way?

**Comment by lucas-0liveira** [comment]
Absolutely, it's definitely possible to set up a custom user role with the capability to upload files when submitting a time sheet, but limit it to a monthly basis. This way, the consultants can attach their invoices.

**Reply by kevinpapst** [reply]
Ok, I am reporting your account to Github now.
